companion-generator/src/main/java/com/gradle/cucumber/companion/generator/CompanionGenerator.java

```diff
@@ -40,25 +42,25 @@ public static void create(CompanionFile companionFile) throws IOException {
                 bw.write("package ");
                 bw.write(companionFile.getPackageName().get());
                 bw.write(";");
-                bw.newLine();
-                bw.newLine();
+                bw.write(NEW_LINE);
```

Comment:
❔ I am not sure I understand this fix. Shouldn't it be `\r\n` on Windows?

Reply:
Is this to avoid changing outputs based on OS?

Reply:
Its to have normalized output, as these files are not really for human consumption and Java is fine `\n`, otherwise we'll have to change the checks in the tests.

Reply:
Thank you for explaining 👍